pandora-client-web/src/components/wiki/wiki.tsx

```diff
@@ -21,6 +25,26 @@ export function Wiki(): ReactElement {
 					<WikiIntroduction />
 				</WikiContent>
 			</UrlTab>
+			<UrlTab name='Rooms' default urlChunk='rooms'>
+				<WikiContent>
+					<WikiRooms />
+				</WikiContent>
+			</UrlTab>
```

Comment:
if we have this pattern for many tabs, I think it is worth lifting it out to a `WikiContentTab`

```tsx
function WikiContentTab({ name, urlChunk = name.toLowerCase(), children }: { name: string urlChunk?: string; children: ReactNodel }): ReactNode {
	return (
		<UrlTab name={ name } default urlChunk={ urlChunk }>
			<WikiContent>
				{ children }
			</WikiContent>
		</UrlTab>
	);
}
```

or something like this